MDANSE/Src/MDANSE/Framework/Jobs/IJob.py

```diff
@@ -156,6 +157,11 @@ def configuration(self):
     def finalize(self):
         if self._log_filename is not None:
             self.remove_log_file_handler()
+        if hasattr(self._outputData, "data_object"):
+            self._in_memory_result = self._outputData.data_object
```

Comment:
Up to you, but could also do:
```suggestion
        self._in_memory_result = getattr(self._outputData, "data_object", None)
```
This will assign a value to `self._in_memory_result`.